wiki/misc/worldtypes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Adding a Custom World Preset
+---
+
+# Adding custom world presets
+If your mod completely overhauls Minecraft's worldgen, you'll want to create a custom worldtype so you give people the option to use your worldgen or MC's vanilla worldgen.
+Now, what is a custom worldtype you may ask? World types define world generation options for Minecraft worlds. They let you change the make-up of your Minecraft worlds by configuring how world generation behaves.
+
+</br>
+</br>
+
+Here is where you specify which dimensions your world preset will apply to. It does require a `minecraft:overworld` dimension or your world preset will not work.
+Valid dimensions are `minecraft:overworld`, `minecraft:nether` and `minecraft:the_end`.
+```json
+"dimensions": {
+  "minecraft:overworld": {
+    "type": "minecraft:overworld"
+    }
+  }
+```
+
+</br>
+</br>
+
+`generator` is where you specify how the worldgen in your world preset behaves. The `type` is a "generator ID" and the following are valid generator IDs:
+```
+noise
+flat
+debug
+```
+
+`biome_source` is where you specify how biomes generate. The `type` parameter here is what kind of biome generation will be used.
```

Comment:
This makes me think that i can add this to a flat or debug world, which is wrong. Make sure you a clear distinction of what fields are supported by what generator types.